roadmap.md

```diff
@@ -1,32 +1,29 @@
 # Roadmap
 
-The MSAL.js team's roadmap is presented as near-term, mid-term, & long-term to give you a better idea of what we are currently working on, what is up next for us, and what we are considering for the future. These are not guaranteed timelines, but instead intended to provide insight into the team's current direction. The roadmap is focused on big features or focus areas; it may not call out each individual work item that is added to the library. While we try to stay consistent with our plans, things can change quickly. Do not write code reliant on these features until they are officially shipped.
+The MSAL.js team's roadmap is presented as In Development, Up Next, and Under Consideration to give you a better idea of what we are currently working on, what is up next for us, and what we are considering for the future. These are not guaranteed timelines or commitments, but instead intended to provide insight into the team's current direction. As an open-source library, you can already view changes the team is actively working on so this roadmap is focused on big features or focus areas; it may not call out each individual work item that is added to the library. While we try to stay consistent with our plans, things can change quickly. Do not write code reliant on these features until they are officially shipped.
```

Comment:
```suggestion
The MSAL.js team's roadmap is presented as _In Development_, _Up Next_, and _Under Consideration_ to give you a better idea of what we are currently working on, what is up next for us, and what we are considering for the future. These are not guaranteed timelines or commitments, but instead intended to provide insight into the team's current direction. As an open-source library, you can already view changes the team is actively working on so this roadmap is focused on big features or focus areas; it may not call out each individual work item that is added to the library. While we try to stay consistent with our plans, things can change quickly. Do not write code reliant on these features until they are officially shipped.
```